components/form_inputs/overflow_buttons/WithChipsOverflowButton.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <v-container>
+    <v-overflow-btn
+      class="my-2"
+      :items="items"
+      label="Overflow Btn - Dense"
```

Comment:
This button is not dense
Please change label

Reply:
Fixed